src/encoded/types/page.py

```diff
@@ -165,12 +148,24 @@ def is_static_page(info, request):
 
 
 def includeme(config):
-    config.add_route(
-        'staticpage',
-        '/*subpath',
-        custom_predicates=[is_static_page],
-        request_method="GET"
-    )
+    with filtered_warnings("ignore", category=DeprecationWarning):
```

Comment:
We're just living with a deprecation warning right now, or we'd be peppered with them in the new pytest. We should fix that sometime, of course.